spec/models/salt_event_handlers/minion_start_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+require "rails_helper"
+
+describe SaltEventHandlers::SaltEvent do
+  let(:salt_event) {
+    event_data = "{\"_stamp\": \"2017-01-24T13:30:20.794326\", "\
```

Comment:
We could avoid all this escaping if we used `event_data = ''`, it's somewhat easier to read. Even plain ruby hashes with a `.to_json`.